docs/kbs_attestation_protocol.md

```diff
@@ -383,20 +383,22 @@ A request for protected resource can fail for three reasons:
 3. The requested resource does not exist. The KBS implementation sends an HTTP
    response with a 404 (`Not Found`) status code.
 
-The KBS protocol currently supports two kinds of resources for an attester to
-request: keys and tokens.
+### Resource Path
 
-### Key Resource
+KBS uses the following path format to locate resources:
 
-Keys are generic protected resources that an authenticated attester can get by
-sending a `GET` HTTP request to the `/kbs/v0/resources/key/<key_id>` endpoint.
+```
+/kbs/v0/resources/<repository>/<type>/<tag>
+```
+
+Here,
 
-The `<key_id>` resource name is defined by the KBS. There must be a different
-`<key_id>` name for each generic secret or resource that the KBS intends to
-deliver to authenticated attesters.
+- `<repository>`: A parent path, can be empty to use the default repository.
+- `<type>`: To distinguish different resource types.
+- `<tag>`: To distinguish different resource instances.
 
```

Comment:
It would be really useful to add a similar comment than https://github.com/confidential-containers/kbs/pull/21/files#r1048155490 here.